kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -75,22 +74,36 @@
         v-model="confirmedPassword"
       />
 
-      <ui-select
-        :name="$tr('selectFacility')"
-        :placeholder="$tr('selectFacility')"
-        :label="$tr('facility')"
-        :value="selectedFacility"
-        :options="facilityList"
-        :invalid="facilityIsInvalid"
-        :error="facilityIsInvalidText"
-        @blur="facilityBlurred = true"
-        @input="updateSelection"
+      <!-- ToS modal only shows up if the box empty prior to clicking -->
+      <!-- Using UI Checkbox for blur events and v-model -->
+      <k-checkbox
+        id="terms-agreement-checkbox"
+        :class="['terms-agreement-checkbox', termsNotAgreed ? 'invalid' : '']"
+        :checked="termsAgreed"
+        @change="showTerms = termsAgreed = $event"
+        @blur="termsAgreementCheckboxBlurred = true"
+        :label="$tr('termsAgreementLabel')"
       />
+      <label
+        v-if="termsNotAgreed"
+        for="terms-agreement-checkbox"
+        class="terms-error-box"
```

Comment:
Needs styling. We can do styling in this branch, just not branding specific like logos and colors.

Reply:
All styling was stripped out and moved to the style PR. It used to be in this branch.

Reply:
But this styling is specific to this instant schools version. This would not exist in the vanilla kolibri vf styled version, right? Unless I am misunderstanding the point of the styling branch.

Reply:
styles are back!